rtic-monotonics/src/nrf/rtc.rs

```diff
@@ -1,96 +1,144 @@
-//! [`Monotonic`] implementation for the nRF Real Time Clocks (RTC).
+//! [`Monotonic`](rtic_time::monotonic::Monotonic) implementation for the nRF Real Time Clocks (RTC).
 //!
 //! # Example
 //!
 //! ```
-//! use rtic_monotonics::nrf::rtc::*;
+//! use rtic_monotonics::nrf::rtc::prelude::*;
+//! nrf_rtc0_monotonic!(Mono);
 //!
 //! fn init() {
 //!     # // This is normally provided by the selected PAC
 //!     # let rtc = unsafe { core::mem::transmute(()) };
-//!     // Generate the required token
-//!     let token = rtic_monotonics::create_nrf_rtc0_monotonic_token!();
-//!
 //!     // Start the monotonic
-//!     Rtc0::start(rtc, token);
+//!     Mono::start(rtc);
 //! }
 //!
 //! async fn usage() {
 //!     loop {
 //!          // Use the monotonic
-//!          Rtc0::delay(100.millis()).await;
+//!          let timestamp = Mono::now();
+//!          Mono::delay(100.millis()).await;
 //!     }
 //! }
 //! ```
 
+/// Common definitions and traits for using the nRF RTC monotonics
+pub mod prelude {
+    pub use crate::nrf_rtc0_monotonic;
+    pub use crate::nrf_rtc1_monotonic;
+    #[cfg(any(feature = "nrf52832", feature = "nrf52833", feature = "nrf52840"))]
+    pub use crate::nrf_rtc2_monotonic;
+
+    pub use crate::Monotonic;
+    pub use fugit::{self, ExtU64, ExtU64Ceil};
+}
+
 #[cfg(feature = "nrf52810")]
-use nrf52810_pac::{self as pac, Interrupt, RTC0, RTC1};
+pub use nrf52810_pac::{self as pac, RTC0, RTC1};
```

Comment:
Making these `pub` is dangerous if we need to bump the PACs. It should be in an `#[doc(hidden)]` module only used to expose them.